### PR TITLE
Archive 2018 and update info for 2019

### PR DESCRIPTION
**Pull request by marialivia16**
The 2019 edition is here! Let's update the website.

I'm planning more of a makeover in a future PR, this is just to update the info.

The old edition will be saved under `/2018/`.

- [x] Add more text (e.g. directions for venue)
- [x] Make sure the sponsor's descriptions are up to date

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -25,10 +25,8 @@
 
       {% if page.title != "Sponsors" %}
       <div class="pt4 tc">
-        <a class="ph2" href="https://www.fundingcircle.com/"><img src="{{ site.url }}/images/sponsors/funding-circle.svg" alt="Funding Circle" width="113" height="50"></a>
+        <a class="ph2" href="https://ovoenergy.com/"><img src="{{ site.url }}/images/sponsors/ovoenergy.png" alt="Ovo Energy" height="50"></a>
```
Comment (cb372) [inline _layouts/default.html]: It's OVO Energy, not Ovo Energy. A common mistake!

File: venue.md [hunk 2]
```diff
@@ -2,26 +2,12 @@
 title: Venue
 ---
 
-LambdAle will be held in the ballroom at [Balham Bowls Club](http://balhambowlsclub.com/).
+LambdAle will be held in the ballroom at [The George IV Pub](https://www.georgeiv.co.uk).
```
Comment (cb372) [inline venue.md]: s/ballroom/Boston Room

File: venue.md [hunk 3]
```diff
@@ -2,26 +2,12 @@
...
-
-Balham Bowls Club is only a couple of minutes' walk from Balham station.
-
-From Balham station, walk north up Balham High Street. Take the first left onto Ramsden Road. The pub is on your left.
-
-<a title="By Ewan Munro from London, UK [CC BY-SA 2.0 (https://creativecommons.org/licenses/by-sa/2.0)], via Wikimedia Commons" href="https://commons.wikimedia.org/wiki/File%3ABalham_Bowls_Club%2C_Balham%2C_SW12_(7047537451).jpg"><img width="512" alt="Balham Bowls Club, Balham, SW12 (7047537451)" src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/ad/Balham_Bowls_Club%2C_Balham%2C_SW12_%287047537451%29.jpg/512px-Balham_Bowls_Club%2C_Balham%2C_SW12_%287047537451%29.jpg"/></a>
-
-Somebody will be at the door to greet you. The conference is in the ballroom upstairs.
-
 ## Accessibility
```
Comment (cb372) [inline venue.md:20]: The venue is on the ground floor and has wheelchair access.

If you have a disability and you would like to discuss your requirements, please get in touch at [info@lambdale.org](mailto:info@lambdale.org). We will do our best to ensure anybody who wants to attend LambdAle can do so.

File: venue.md [hunk 4]
```diff
@@ -2,26 +2,12 @@
...
+The address is [185 High Road, Chiswick, London, W4 2DR](https://goo.gl/maps/2kHAdvHpHqJ2).
 
-## Map
-
-<iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d19893.772141396432!2d-0.152503!3d51.444908!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x4928317423696e4d!2sBalham+Bowls+Club!5e0!3m2!1sen!2suk!4v1518043181690" width="300" height="300" frameborder="0" style="border:0" allowfullscreen></iframe>
+<iframe src="https://www.google.com/maps/embed?pb=!4v1550427124320!6m8!1m7!1sCAoSLEFGMVFpcE8yekNEajNfWlJDUjM5OXV4OXZrMWpjX2R0dkdUQ00zekd5cXI0!2m2!1d51.49242125220601!2d-0.2570486842103037!3f168.99!4f8.120000000000005!5f0.4000000000000002" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
 
 ## Directions
 
-From central London, either take the Northern line southbound or take National Rail from London Victoria towards Sutton (usually platform 1 or 2).
```
Comment (cb372) [inline venue.md:15]: The nearest tube stop is Turnham Green.

From central London, take the District line westbound towards Ealing Broadway or Richmond.

Turn left out of Turnham Green station and walk to the end of the road. Turn right and walk down Chiswick High Street for about 100 metres. The pub is on the left hand side.

Somebody will be at the door to greet you. The conference is in the Boston room at the back of the pub.

**Review by cb372 — COMMENTED** [review]
Looks good! Thanks for updating.

File: index.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
 LambdAle is a functional programming conference in a pub.
 
-**Date:** Saturday 1st September
+**Date:** Saturday 21st September
 
-**Location:** Balham Bowls Club, Balham, London
+**Location:** George IV in Chiswick, West London
 
-**Content**: 11 half-hour talks, with an FP-flavoured pub quiz to finish.
+**Content**: 10 hours of talks, with an FP-flavoured pub quiz to finish.
```
Comment (cb372) [inline index.md]: 10 hours?! How about "A whole day of exciting talks"

File: venue.md (hunk 2 above)
Comment (Qthunder) [reply venue.md]: I'm not sure what this means - couldn't find a link to the specific room on their website.

Comment (cb372) [reply venue.md]: Their largest function room is called the Boston Room. There's a reference to it on the ["360 tour" page](https://www.georgeiv.co.uk/360-tour).

**Review by cb372 — APPROVED** [review]
Thanks!

**Pull request merged**